api/src/utils/csv-utils/row-validators/taxon-row-validator.ts

```diff
@@ -27,6 +27,10 @@ export const getTaxonRowValidator = <StaticHeaderType extends Uppercase<string>
     const taxonHeader = utils.getWorksheetHeader(taxonStaticHeader, params.row);
 
     if (taxonIdentifierCell === undefined) {
+      if (utils.isCellOptional(taxonStaticHeader)) {
+        return [];
+      }
+
```

Comment:
Let me know if this addition, and the related utils method, is okay or not. Or if there is a better way to handle this case, etc.

Reply:
From our discord discussion:
The optional property for a standard column of the CSVConfig just indicates that the column can be omitted from the CSV. Different than allowing undefined / empty values but the header must be defined still.

I think all cases currently where we allow undefined values we also allow omitting the header (optional). For future proofing, we can provide a property to this row-validator to allow undefined values, seperate from the column optional logic.

Reply:
Updated.